getting-started/templates/systemlink-values.yaml

```diff
@@ -355,9 +355,13 @@ dashboardhost:
 
     ## This sets the GF_INSTALL_PLUGINS env variable
     ## https://grafana.com/docs/grafana/latest/installation/docker/#install-plugins-from-other-sources
+    ## Uncomment to configure additional plugins. This will override the plugins included by default
+    ## so ensure you update with the latest versions listed in the default values file.
     ##
-    plugins:
-      - http://localhost:8080/ni/plugins/systemlink-notebook-datasource/1.1.0.zip;ni-slnotebook-datasource
+    # plugins:
+    #   - http://localhost:8080/ni/plugins/systemlink-notebook-datasource/X.X.X.zip;ni-slnotebook-datasource
```

Comment:
Does every customer have to do this? Filling out the X.X.X.X seems like a poor experience. Can we help this in any way?

Reply:
No, customers would only do this if they want to customize Grafana with additional plugins than the ones we ship with.

Reply:
Is there an easy way to understand what we ship? Is this them?

Reply:
Yes, this commented stuff is meant to represent what we ship, but we were hoping to not have to come back and update the comments every time we bump versions.

Reply:
Perhaps we should create help content with an example of how to add a new plugin to a Grafana deployment? Not urgent.

Reply:
If we think it's more confusing than helpful in its current form, we could also remove this commented plugin stuff until it's better documented.

Reply:
Why wouldn't we have these values set as defaults such that the user doesn't need to specify them at all in the top level chart? Would this also address the concern about doc churn to update this file each time the plugin versions are bumped?

Reply:
These are set as defaults in the chart we provide to customers. They would only need to uncomment this if they want to add additional Grafana plugins beyond what we ship. In that scenario, they would also need to specify the ones that we ship in addition to the ones they want to add.

Reply:
@MelissaHilliard - What are your thoughts on this? Could you help with the wording of the comment maybe? Or should this documentation go somewhere else? JD also suggested above that we create help content with an example of this process, thoughts? Let me know if you'd prefer we have a quick chat to discuss what this means for the customer.

Reply:
@cameronwaterman - If I understand the situation correctly, the comment might read, "To include additional Grafana plugins, uncomment this section and update the URL where X.X.X.zip is the file you want to reference."

Reply:
@MelissaHilliard - They would need to update the X.X.X.zip to the latest version of those plugins (they are plugins that we ship with SLE), and then can add additional plugins to the list for new ones they want to add.

Reply:
@cameronwaterman So maybe, "To include additional Grafana plugins, uncomment this section, update the existing URLs where X.X.X.zip is the latest version, and add the URLs for any additional plugins."